esx_service/cli/vmdkops_admin.py

```diff
@@ -563,10 +563,13 @@ def status(args):
 
 
 def set_vol_opts(args):
-    set_ok = vmdk_ops.set_vol_opts(args.volume, args.options) 
-    if set_ok:
-        print('Successfully updated settings for : {0}'.format(args.volume))
-    else:
+    try:
+        set_ok = vmdk_ops.set_vol_opts(args.volume, args.options) 
+        if set_ok:
+           print('Successfully updated settings for : {0}'.format(args.volume))
+        else:
+           print('Failed to update {0} for {1}.'.format(args.options, args.volume))
+    except Exception as ex:
         print('Failed to update {0} for {1}.'.format(args.options, args.volume))
```

Comment:
+1 on this. 
`WARNING:root:Invalid options: ['acces']` seems to be showing in the log, not sysout. Or is it in sysout ? 
Can you clarify the exact print happening on typos here?